src/components/Fieldset/Fieldset.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames/bind';
+import styles from './Fieldset.scss';
+const cx = classnames.bind( styles );
+
+const Fieldset = ( {
+	autoComplete,
+	defaultValue,
+	describedBy,
+	disabled,
+	isMultiline,
+	inputRef,
+	id,
+	label,
+	maxLength,
+	placeholder,
+	onBlur,
+	onChange,
+	onClick,
+	onFocus,
+	onInvalid,
+	pattern,
+	readOnly,
+	required,
+	type,
+} ) => {
+	// ClassNames used by both <input> and <textarea>.
+	const sharedClassNames = {
+		disabled,
+		readOnly,
+		[type]: type,
+	};
+
+	const requiredField = label && required;
+
+	return (
+		<fieldset className={styles.container}>
+			<label className={`${styles.label} ${styles.requiredLabel}`}>
+				{label}
+				{requiredField && <span className={styles.requiredAsterisk}> * </span>}
+				{isMultiline &&
+					<textarea
+						className={cx( 'textarea', { ...sharedClassNames } )}
+						rows={6}
+						aria-describedby={describedBy}
+						autocomplete={autoComplete}
+						defaultValue={defaultValue}
+						describedBy={describedBy}
+						disabled={disabled}
+						id={id}
+						maxLength={maxLength}
```

Comment:
should this be `maxlength`? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea